shell/platform/common/client_wrapper/include/flutter/encodable_value.h

```diff
@@ -197,9 +197,10 @@ class EncodableValue : public internal::EncodableValueVariant {
   // other types, std::monostate uses aren't self-documenting.
   bool IsNull() const { return std::holds_alternative<std::monostate>(*this); }
 
-  // Convience method to simplify handling objects received from Flutter where
-  // the values may be larger than 32-bit, since they have the same type on the
-  // Dart side, but will be either 32-bit or 64-bit here depending on the value.
+  // Convenience method to simplify handling objects received from Flutter
```

Comment:
GitHub's diff makes this look like a reformat, but the fix was in the first word.